vhdbuilder/packer/components.json

```diff
@@ -168,7 +168,7 @@
       "multiArchVersions": [
         "v3.20.0",
         "v3.21.0",
```

Comment:
If tigera v1.23.1/calico v3.21.0 is not used on AKS RP any more, can we remove them from vhd caching. In last official weekly amd64 linux vhd build, a few vhd build failed because disk is full. Ganesh cleaned up a few images in order that the vhd build can pass.

Reply:
We can also remove v3.20.0 since it is no longer in use anywhere. Though I think we like to stick with the current one planned for release and also the one before it, in case of revert.